openresty-patches/patches/1.15.8.2/nginx-1.15.8_01-upstream_client_certificate_and_ssl_verify.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/nginx-1.15.8/src/http/ngx_http_upstream.c b/nginx-1.15.8/src/http/ngx_http_upstream.c
+index 90710557..539a4db9 100644
+--- a/nginx-1.15.8/src/http/ngx_http_upstream.c
++++ b/nginx-1.15.8/src/http/ngx_http_upstream.c
+@@ -8,6 +8,9 @@
+ #include <ngx_config.h>
+ #include <ngx_core.h>
+ #include <ngx_http.h>
++#if (NGX_HTTP_LUA_KONG)
++#include <ngx_http_lua_kong_module.h>
++#endif
+
+
+ #if (NGX_HTTP_CACHE)
+@@ -1694,7 +1694,13 @@
+     c->sendfile = 0;
+     u->output.sendfile = 0;
+
++
++#if (NGX_HTTP_LUA_KONG)
++    if (u->conf->ssl_server_name ||
++	    ngx_http_lua_kong_get_upstream_ssl_verify(r, u->conf->ssl_verify)) {
```

Comment:
Style: when if spans multiple lines, it should be written as:

```
if (u->conf->ssl_server_name
    || ngx_http_lua_kong_get_upstream_ssl_verify(r, u->conf->ssl_verify))
{
```

See: http://lxr.nginx.org/source/src/http/modules/ngx_http_gzip_filter_module.c#0249 for an example